### PR TITLE
ANDROID-10301 Fix link button in cards

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
ANDROID-10301 The link button is not shown if the action is not present

### :construction: How do we do it?
Replacing and `&&` by an `||` and adjusting the padding

### ☑️ Checks
- [ ] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [ ] Tested with dark mode.
- [ ] Tested with API 21.

### :test_tube: How can I test this?
image

### CODE REVIEW

**Review by dagonco — APPROVED** [review]
Thanks @gmerinojimenez !

File: library-compose/src/main/java/com/telefonica/mistica/compose/card/Card.kt [hunk 1]
```diff
@@ -54,7 +55,7 @@ fun Card(
 
 @Composable
 internal fun CardActions(primaryButton: Action?, linkButton: Action?) {
-    if (primaryButton != null && linkButton != null) {
+    if (primaryButton != null || linkButton != null) {
```
Comment (yamal-alm) [inline library-compose/src/main/java/com/telefonica/mistica/compose/card/Card.kt:58]: does non-compose component have the same condition?

**Review by yamal-alm — APPROVED** [review]
thanks!

File: library-compose/src/main/java/com/telefonica/mistica/compose/card/Card.kt (hunk 1 above)
Comment (gmerinojimenez) [reply library-compose/src/main/java/com/telefonica/mistica/compose/card/Card.kt:58]: yes, the && was my fault

**Pull request merged**